tutorial/tests/testsuite.py

```diff
@@ -46,7 +45,7 @@ def ipytest(self, line, cell):
         function_names = {}
         for name, function in self.shell.user_ns.items():
             if name.startswith("solution_") and callable(function):
-                function_names[name.lstrip("solution_")] = function
+                function_names[name[9:]] = function  # Remove the `solution_` prefix
```

Comment:
Since we're using Python 3.9+ (hopefully), I think `name.removeprefix('solution_')` is even better than slicing

Reply:
Indeed, it is more explicit 👍